File: week1/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask user for height
    int height;
    do
    {
        height = get_int("Height (between 1 and 8 inclusive): \n");
    }
    while (height < 1 || height > 8);

    // create a pyramid
    for (int line = 0; line < height; line++)
    {
        for (int j = 0; j < height - line - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < line + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/pset6/sentimental-mario-more/mario.py
from cs50 import get_int

# TODO
height = get_int("Height: ")

while True:
    if height < 1 or height > 8:
        height = get_int("Height: ")
    else:
        break

for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for j in range(i):
        print("#", end="")
    print("#")